src/repositories/request-repository.ts

```diff
@@ -18,6 +19,9 @@ const ANCHOR_DATA_RETENTION_WINDOW = 1000 * 60 * 60 * 24 * 30 // 30 days
 export const MAX_ANCHORING_DELAY_MS = 1000 * 60 * 60 * 12 //12H
 export const PROCESSING_TIMEOUT = 1000 * 60 * 60 * 3 //3H
 export const FAILURE_RETRY_WINDOW = 1000 * 60 * 60 * 48 // 48H
+const TRANSACTION_MUTEX_ID = 4532
```

Comment:
any reason for this value?  Totally arbitrary?

Reply:
yes totally arbitrary haha

Reply:
Can we add a comment or make it like `1234` so it's more clearly arbitrary?